docs/core/diagnostics/dotnet-gcdump.md

```diff
@@ -0,0 +1,110 @@
+---
+title: dotnet-gcdump - .NET Core
+description: Installing and using the dotnet-gcdump command-line tool.
+ms.date: 07/26/2020
+---
+# Heap analysis tool (`dotnet-gcdump`)
+
+**This article applies to:** ✔️ .NET Core 3.1 SDK and later versions
+
+## Installing `dotnet-gcdump`
+
+To install the latest release version of the `dotnet-gcdump` [NuGet package](https://www.nuget.org/packages/dotnet-gcdump), use the [dotnet tool install](../tools/dotnet-tool-install.md) command:
+
+```dotnetcli
+dotnet tool install -g dotnet-gcdump
+```
+
+## Synopsis
+
+```console
+dotnet-gcdump [-h|--help] [--version] <command>
+```
+
+## Description
+
+The `dotnet-gcdump` global tool is a way to collect gcdumps of live .NET processes. It is built using the EventPipe technology which is a cross-platform alternative to ETW on Windows. Gcdumps are created by triggering a GC in the target process, turning on special events, and regenerating the graph of object roots from the event stream. This allows for gcdumps to be collected while the process is running with minimal overhead. These dumps are useful for several scenarios:
+
+- comparing the number of objects on the heap at several points in time.
+- analyzing roots of objects (answering questions like, "what still has a reference to this type?").
+- collecting general statistics about the counts of objects on the heap.
+
+## Options
+
+- **`--version`**
+
+  Displays the version of the `dotnet-gcdump` utility.
+
+- **`-h|--help`**
+
+  Shows command-line help.
+
+## `dotnet-gcdump collect`
```

Comment:
A Commands section here like the one in https://raw.githubusercontent.com/dotnet/docs/master/docs/core/diagnostics/dotnet-counters.md might be helpful. Then the following h2s would correspond to the commands in the table.

Reply:
@tdykstra, The live docs has a side-bar containing links to all H2s, I personally think a commands section with only links to the heading is noisy and repetitive with the sidebar.

Reply:
OK to omit the Commands section.